Gems/Atom/Tools/ShaderManagementConsole/Code/Source/Document/ShaderManagementConsoleDocument.cpp

```diff
@@ -244,6 +244,11 @@ namespace ShaderManagementConsole
         return true;
     }
 
+    bool ShaderManagementConsoleDocument::CanSave() const
+    {
+        return true;
+    }
+
```

Comment:
This should be fine now. Previously it would not let you save without opening because the document path would not be set. But I changed it to do you save as if the document was not previously saved.